doc/doc_build_and_repository_info.md

```diff
@@ -1,15 +1,16 @@
-### **CPU only mode**
+## Build and Repository Info
 
-CPU mode enabled by arg --cpu-only for all stages. Follow requirements-cpu.txt to install req packages.
-Do not use DLIB extractor in CPU mode, it's too slow.
-Only H64 or SAE (with low settings) models reasonable to train on home CPU.
+DeepFaceLab officially supports Windows-only. Linux and OS X are untested.
```

Comment:
If you want to support mac/linux/docker - create fork, it will be referenced here.